integration-tests/sanity-functional/src/main/resources/config/common/webdriver.properties

```diff
@@ -0,0 +1,48 @@
+#
+# AET
+#
+# Copyright (C) 2013 Cognifide Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+webdriver.defaultTimeout=10
+page.title.timeout=30
+thread.count=3
+
+
+# +++ Chrome +++
+#webdriver.chrome.driver property should point to Chrome webdriver executable.
+webdriver.type=chrome
+webdriver.chrome.driver=D:/Selenium/chromedriver.exe
```

Comment:
When this path is required. Will this work on all instances?

Reply:
You are right: this is an arbitrary value. If chromedriver is not found then there is a stacktrace with following error:

```
2) Error in custom provider, java.lang.IllegalStateException: The driver executable does not exist: D:\Selenium\chromedriver.exe
```

If this config value is commented then we have following error:

```
2) Error in custom provider, java.lang.IllegalStateException: The path to the driver executable must be set by the webdriver.chrome.driver system property;
```

What should we do here? Currently on master we have: 

* [integration-tests/sanity-functional/src/main/config/webdriver.properties#L23](https://github.com/Cognifide/aet/blob/master/integration-tests/sanity-functional/src/main/config/webdriver.properties#L23)

Reply:
Maybe we should at least document the necessary change on the path to the chromedriver in the comment on in the [it-tests README](https://github.com/Cognifide/aet/tree/master/integration-tests#sanity-functional) ?

Reply:
I have update the README